ui/app/templates/components/clients/dashboard.hbs

```diff
@@ -175,26 +175,29 @@
       </EmptyState>
     {{/if}}
   {{/if}}
+</div>
 
-  {{! BILLING START DATE MODAL }}
-
-  <Modal
-    @title="Edit start month"
-    @onClose={{action (mut this.showBillingStartModal) false}}
-    @isActive={{this.showBillingStartModal}}
-    @showCloseButton={{true}}
-  >
-    <section class="modal-card-custom has-padding-m">
+{{! BILLING START DATE MODAL }}
+{{#if this.showBillingStartModal}}
```

Comment:
![dd-two](https://github.com/hashicorp/vault/assets/68122737/93e8d6fc-c674-40ba-b3a3-684831dc6ad1)

Reply:
padding has changed since gif was taken
<img width="500" alt="Screenshot 2023-10-04 at 12 58 49 PM" 
src="https://github.com/hashicorp/vault/assets/68122737/5c2cde74-d5da-4e17-b741-ffdb5ba2fa2c">